acos_client/v30/slb/virtual_server.py

```diff
@@ -45,19 +46,19 @@ def _set(self, name, ip_address=None, status='stats-data-enable',
 
         return self._post(self.url_prefix + name, params, **kwargs)
 
-    def create(self, name, ip_address, status='stats-data-enable', **kwargs):
+    def create(self, name, ip_address, status='stats-data-enable', arp_disable=None, **kwargs):
         try:
             self.get(name)
         except acos_errors.NotFound:
             pass
         else:
             raise acos_errors.Exists
 
-        return self._set(name, ip_address, status, **kwargs)
+        self._set(name, ip_address, status, arp_disable=arp_disable, **kwargs)
```

Comment:
Why do we stop returning the created virtual server. We need it (#110) to record sflow uuids.